scripts/analytics/addon_snapshot.py

```diff
@@ -84,11 +84,24 @@ def get_events(self, date=None):
             connected_count = 0
             deleted_count = 0
             disconnected_count = 0
+            connected_public_count = 0
+            connected_private_count = 0
+            connected_public_affiliated_count = 0
+            connected_private_affiliated_count = 0
             node_settings_model = addon.models.get('nodesettings')
             if node_settings_model:
                 for node_settings in paginated(node_settings_model):
-                    if node_settings.owner and not node_settings.owner.all_tags.filter(name='old_node_collection', system=True).exists():
+                    node = node_settings.owner
+                    if node and not node_settings.owner.all_tags.filter(name='old_node_collection', system=True).exists():
                         connected_count += 1
+                        if node.is_public:
+                            connected_public_count += 1
+                            if node.affiliated_institutions.exclude(name='Center For Open Science [Test]').exists():
```

Comment:
questions/nits:
1) Why just adding +1 here instead of the total count of affiliated institutions. Just counting the number of nodes with affiliates?
2) Could do this check before this branch since it happens no matter what, then increment if True
`connected_[public|private]_affiliated_count += 1 if affiliated_institutions else 0`

Reply:
1. It's plus one because the metrics requested are 'number of public/private nodes with addons connected which are affiliated with a non-COS institution (https://openscience.atlassian.net/browse/REQS-159)
2. You're right, I'll modify that.